playbooks/openshift/cinder-registry.yml

```diff
@@ -0,0 +1,19 @@
+--- 
+# Create a volume for the registry
+- hosts: localhost
+  roles: 
+  - infra-ansible/roles/osp/admin-volume
+
+- hosts: OSEv3
+  tasks:
+  - name: "set registry volumeID fact"
+    set_fact: 
+      openshift_hosted_registry_storage_openstack_volumeID: "{{ disk.volume.id }}"
```

Comment:
Per comment above - just add `openshift_hosted_registry_storage_volume_size: "{{ disk.size }}Gi"` here

Reply:
... there may be a better fact to use than `disk.volume.id`, but you probably have the capture so you can decide which one it is.

Reply:
Yup, `disk.volume.size`

Reply:
Thx. Code change looks good. Still need to update the doc and example inventory. Could be good to call-out that these two facts will be automatically set.

Reply:
Docs updated -- PTAL